src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,16 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery extends ColorSupplier {
+    public Ball getRandomBall() {
+
+        Ball ball = new Ball();
+
+        ball.setBallColor(ColorSupplier.getRandomColor());
+
+        ball.setBallNumber(new Random().nextInt(100));
+
+        return ball;
+    }
+}
```

Comment:
remove empty lines and create objects in class and use it a methods 
```suggestion
public class Lottery {
    public static final int NUMBER = 100;
    private ColorSupplier colorSupplier = new ColorSupplier();
    private Random random = new Random();

    public Ball getRandomBall() {
        Ball ball = new Ball();
        ball.setBallColor(colorSupplier.getRandomColor());
        ball.setBallNumber(random.nextInt(NUMBER));
        return ball;
    }
}
```